setup.py

```diff
@@ -46,8 +46,8 @@
             "upgrade PyPy to use this library."
         )
 else:
-    requirements.append("cffi>=1.1.0")
-    setup_requirements.append("cffi>=1.1.0")
+    requirements.append("cffi>=1.7.0")
```

Comment:
Bumping the cffi requirement to require the latest without a reason (e.g. we need a feature only provided by a newer version) is not something we want to do.